azbus/disposition.go

```diff
@@ -0,0 +1,147 @@
+package azbus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus"
+	"github.com/datatrails/go-datatrails-common/tracing"
+)
+
+type Disposition int
```

Comment:
Minor: I relate disposition to ones mood? i.e. they have a cheery disposition.

Can we have a comment here to describe what this is intending to convey?

Reply:
I have added comment.

azure-sdk-for-go refers to these internally as DispositionStatus and alternatively as Settlement - Settlement seemed clumsily named.